format_spec/fragment.md

```diff
@@ -186,6 +186,17 @@ The fragment min max sum null count is a [generic tile](./generic_tile.md) with
 | Sum | `uint64_t` | Sum value for attribute/dimension N |
 | Null count | `uint64_t` | Null count value for attribute/dimension N |
 
+The processed conditions is a [generic tile](./generic_tile.md) with the following internal format:
```

Comment:
Documentation point: Please add a few sentences explaining _what_ a processed condition represents.

Reply:
Also: what is the ordering of these values? As-applied?